internal/components/components_test.go

```diff
@@ -26,6 +26,7 @@ import (
 
 func TestDefaultComponents(t *testing.T) {
 	expectedExtensions := []string{
+                "headerssetterextension"
```

Comment:
```suggestion
                "headerssetterextension",
```
Also, please add it in alphabetical order.